src/shared/scrapers/USA/MO/st-louis-county.js

```diff
@@ -0,0 +1,34 @@
+import * as fetch from '../../../lib/fetch/index.js';
+import * as parse from '../../../lib/parse.js';
+import maintainers from '../../../lib/maintainers.js';
+import * as geography from '../../../lib/geography/index.js';
+
+const scraper = {
+  county: 'St. Louis County',
+  state: 'MO',
+  country: 'USA',
+  aggregate: 'county',
+  priority: 1,
+  sources: [
+    {
+      url: 'https://stlouisco.com/Your-Government/County-Executive/COVID-19',
+      name: 'St. Louis County COVID-19 page'
+    }
+  ],
+  url: 'https://stlcogis.maps.arcgis.com/apps/MapSeries/index.html?appid=6ae65dea4d804f2ea4f5d8ba79e96df1',
```

Comment:
This data is not present in the other data source?

Reply:
This is the authoritative source for St. Louis County.  There is a discrepancy in the data from the MO Health Dept and the county website.  I believe the county dataset is updated more frequently making it a prefered source.  

County Arc Map: https://stlcogis.maps.arcgis.com/apps/MapSeries/index.html?appid=6ae65dea4d804f2ea4f5d8ba79e96df1

Health Dept Map: https://www.arcgis.com/apps/opsdashboard/index.html#/1fd32012487c4e5c936a13389f9731c3

Reply:
Including CSV format:
county: https://opendata.arcgis.com/datasets/4b76809cf8084f87b5ee4e18c33c9d29_0.csv
state: https://opendata.arcgis.com/datasets/6f2a47a25872470a815bcd95f52c2872_0.csv

Notice the "edit date" columns of these

Reply:
Excellent. Thank you